app/controllers/metrics/metrics.js

```diff
@@ -0,0 +1,57 @@
+const mongo = require('../../lib/mongo');
+const cache = require('../../lib/cache');
+const logger = require('../../lib/logger');
+
+function getBadgeInDatabase(badgeId) {
```

Comment:
Something like `getBadgeCount()` would be more suitable, as this function does not fetch the actual badge.